### ISSUE TITLE
Angular2 CellRenderer: params.value is not getting updated

### ISSUE THREAD

**Issue by stephanjust** [ceolter/ag-grid-ng2#26]
The new Angular2 CellRenderer functionality somehow doesn't update ```params.value``` when recreating cells after editing.

Old CellRenderer:
```
export class CellRenderer {
    init(params) {
        console.log(this.params.value)
        console.log(this.params.data.reklamationsgrund);
    }
}
```
with ```reklamationsgrund``` being the field that I changed.
Output in my case after changing values twice:
```
DatentypWert_366300005
DatentypWert_366300005

DatentypWert_366300006
DatentypWert_366300006
```

New Angular CellRenderer:
```
export class AngularCellRenderer implements AgAware {
    public agInit(params): void {
        console.log(this.params.value);
        console.log(this.params.data.reklamationsgrund);
    }
}
```
Output in my case after changing values twice:
```
DatentypWert_366300005
DatentypWert_366300005

DatentypWert_366300005
DatentypWert_366300006
```

This means the original row is getting updated properly (I can confirm this as the CellEditor is a dropdown menu, and editing it again has the correct field pre-selected) but ```params.value``` is not (no matter how often I change the value, updates are not being reflected).

**Comment by seanlandsman** [comment]
Hi - this has now be fixed and will be available in the next release

thanks for reporting it

**Comment by ceolter** [comment]
i've just released ag-grid-ng2 5.2.3 with this fix.
